node-hub/feetech-client/README.md

```diff
@@ -24,17 +24,7 @@ nodes:
 
     env:
       PORT: COM9 # e.g. /dev/ttyUSB0 or COM9
-      
-      IDS: 1 2 3 4 5 6
-      JOINTS: shoulder_pan shoulder_lift elbow_flex wrist_flex wrist_roll gripper
-      MODELS: scs_series scs_series scs_series scs_series scs_series scs_series
-
-      TORQUE: True True True True True True
-
-      INITIAL_GOAL_POSITION: None None None None None None
-
-      OFFSETS:  -2048 2048 2048 2048 1024 2048
-      DRIVE_MODES:  POS NEG NEG NEG NEG NEG
+      CONFIG: config.json # the configuration file for the motors
```

Comment:
Could we provide either a default config or a template config ? So that people knows what it looks like

Reply:
I'm adding a `config.template.json` file inside the node, and I'm explaining the parameters in the README